cli/tools/test_runner.rs

```diff
@@ -450,6 +464,7 @@ pub async fn run_tests(
   let test_options = json!({
       "disableLog": quiet,
       "filter": filter,
+      "shuffle": shuffle,
```

Comment:
Why is the second shuffle necessary in the JS code?

Reply:
Shuffle of test definitions as-well as modules.